README.md

```diff
@@ -56,3 +57,34 @@ And then run the image using
 
 ```docker container run  -d -p 3000:3000 laa-apply-for-legalaid-api```
 
+
+## Postgres Sql
```

Comment:
I think these instructions currently cover two things:

1. bootstrapping project from scratch in dev
2. migrating from current sqlite to postgres

I think in the README, we should have just 1), maybe we have a "getting started" section for that. Put 2) in the PR description and point people at it in slack. It's just a one-off thing.

Also, I think it'll be simpler if we make a strong opinion about how the team *should* run postgres, because it makes the README easier to follow. If people really want to run a local non-Docker postgres, then we can tuck the instructions to do that away somewhere.

Reply:
Lets wait for a few other opinions on this first infact i want everyone to help write this bit up.  My aim was to provide a base sert of instructions which can get everyone going.

Reply:
Outcome from our real-life discussion:

- prefer getting started & how to run sections with a list of all the commands needed to get things up and running
- you're going to do a future PR to condense them down to a `make command` anyway, so we don't need to fix anything until then